playwright.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig({
   /* Retry on CI only */
   retries: process.env.CI ? 2 : 0,
   /* Opt out of parallel tests on CI. */
-  workers: process.env.CI ? 1 : undefined,
+  workers: process.env.CI ? 1 : 1,
```

Comment:
How about we limit number of workers only in headed mode? since GHA test seems to be fine (not flaky) I would prefer to keep running tests in parallel

Reply:
Oki, makes sense. :)

Reply:
Fixed